hyperspy/_signals/eels.py

```diff
@@ -287,12 +287,13 @@ def estimate_zero_loss_peak_centre(s, mask, signal_range):
                 else self.axes_manager[-1].axis[0])
         right = (right if right < self.axes_manager[-1].axis[-1]
                  else self.axes_manager[-1].axis[-1])
-        self.align1D(
-            left,
-            right,
-            also_align=also_align,
-            show_progressbar=show_progressbar,
-            **kwargs)
+        if self.axes_manager.navigation_size > 1:
+            self.align1D(
+                left,
+                right,
+                also_align=also_align,
+                show_progressbar=show_progressbar,
+                **kwargs)
```

Comment:
What does the rest of the group think? I feel like most people would expect `s.align_zero_loss_peak(True)` to set the zero loss peak at zero independently of the shape of the signal? I don't see a reason why this behaviour should *not* be included.